.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: dart
+dart:
+  - stable
+script:
+  - pub run dart_dev format --check
+  - pub run dart_dev analyze
+  - pub run dart_dev test
+  - pub run dart_dev coverage --no-html
```

Comment:
Will do. This was copypasta'd [from w_module](https://github.com/Workiva/w_module/blob/master/.travis.yml#L8), so that needs to be updated as well.